docs/calendar.css

```diff
@@ -361,6 +374,11 @@ corresponding value from the list below*/
     float: right;
   }
   
+  #forkMe {
```

Comment:
Oh I see. The diff on GitHub obscures the fact that this is inside of a `@media` query.